Assets/Scripts/Player/Player.cs

```diff
@@ -0,0 +1,45 @@
+using UnityEngine;
+using UnityEngine.Events;
+using IJunior.TypedScenes;
+
+public class Player : MonoBehaviour
+{
+    public event UnityAction<int> CoinsAmountChanged;
```

Comment:
https://stackoverflow.com/questions/150479/order-of-items-in-classes-fields-properties-constructors-methods